features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -80,16 +95,42 @@ fun RoomMemberListView(
                 .padding(padding),
             verticalArrangement = Arrangement.spacedBy(16.dp),
         ) {
-            UserListView(
-                state = state.userListState,
-                onUserSelected = ::onUserSelected,
-            )
+            Column {
+                RoomMemberSearchBar(
+                    query = state.searchQuery,
+                    state = state.searchResults,
+                    active = state.isSearchActive,
+                    placeHolderTitle = stringResource(StringR.string.common_search_for_someone),
+                    onActiveChanged = { state.eventSink(RoomMemberListEvents.OnSearchActiveChanged(it)) },
+                    onTextChanged = { state.eventSink(RoomMemberListEvents.UpdateSearchQuery(it)) },
+                    onUserSelected = ::onUserSelected,
+                    modifier = Modifier.fillMaxWidth()
+                )
+            }
 
-            if (!state.userListState.isSearchActive) {
-                if (state.allUsers is Async.Success) {
+            if (!state.isSearchActive) {
+                if (state.roomMembers is Async.Success) {
```

Comment:
maybe extract this content into a dedicated method "RoomMembersSearchResults", wdyt?

Reply:
Oh, this is not the search result here, so `RoomMembersList`?